src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs

```diff
@@ -19,6 +19,10 @@ public class CleanseLogMessage
                 new Regex(@"/fetch/[a-z0-9]{32}/(?<secret>[a-z0-9]{32})", RegexOptions.Compiled),
                 new Regex(@"getnzb.*?(?<=\?|&)(r)=(?<secret>[^&=]+?)(?= |&|$)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
                 
+                // Trackers Announce Keys; Designed for Qbit Json; should work for all in theory
+                new Regex(@"(?<=announce/)(?<secret>[a-z0-9]{16,})", RegexOptions.Compiled | RegexOptions.IgnoreCase),
```

Comment:
No need for a lookaround, only the 'secret' group is replaced.
I'm fine with having a single one too: `@"announce(\.php)?/(?<secret>[a-z0-9]{16,})|(?<secret>[a-z0-9]{16,})/announce"`